configs/sim/axis/vismach/puma/puma_cube.txt

```diff
@@ -8,6 +8,6 @@ Usage:
   Ctrl-Home -- Home All
   R ---------- Run
 
-Example gocde (.ngc) files are:
+Example gcode (.ngc) files are:
```

Comment:
"G-code"?

Reply:
It's not uncommon to see this 'abbreviation', would you like me to change it ?

Reply:
In the documentation we have changed all variants of G-code/Gcode/g-code/gcode/... to G-code. Have not touched anything (almost) outside docs, but somehow I feel that it would be nice to be consistent throughout the code base. Funnily enough, not even the standard is consistent in the spelling of G-code, so it is a non-issue, but the world should see that we care for such details.

Reply:
Roger that, makes sense. Done in [10233c8](https://github.com/LinuxCNC/linuxcnc/pull/2133/commits/10233c8bcce2f61d1655e7ae2aa8b5718964a4a6)